docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md

```diff
@@ -0,0 +1,17 @@
+# How to remotely configure IoT devices?
+
+With IoT devices enabled with LwM2M protocol, you can use a set of standardized remote operations to configure your IoT devices remotely.
```

Comment:
Twice `IoT Devices`, also `remote operations` to configure `remotely` sound weird too.